src/components/TableCreator/tableCreator.tsx

```diff
@@ -95,7 +88,7 @@ export const TableCreator = <T extends RowData>({
       ),
     [columns]
   );
-  const initialState = getInitialState(columns, defaultSort);
+  const initialState = getInitialState(columns);
```

Comment:
Removing `defaultSort` from `initialState` as this should now be defined in one place in a TanStack API format under `tableOptions.intialState.sorting`.